.github/workflows/build.yml

```diff
@@ -15,31 +15,31 @@ jobs:
     - name: Build (AArch64)
       run: |
         ANDROID_NDK_HOME=./android-ndk-r23b CXX=aarch64-linux-android31-clang++ ./build.sh
-        zip -r ih8sn-aarch64.zip 60-ih8sn.sh ih8sn ih8sn.conf ih8sn.rc push.ps1 push.sh
+        zip -r ih8sn-aarch64.zip META-INF system push.ps1 push.sh
```

Comment:
This will put all user configs in flashable zip, not sure if this is a good idea...

Reply:
Releases are done on a fresh clone right?.

Reply:
I'm mostly referring to local builds here.